docs/api/README.md

```diff
@@ -18,6 +18,7 @@ API Specification:
 
 Current transfer adapters include:
   * [Basic](./basic-transfers.md)
+  * [Static](./static-transfers.md)
```

Comment:
I'd like to leave this document in the `proposals` directory until it's implemented, since otherwise folks might think we support this when we don't yet.

Reply:
Sure.